ADQL.tex

```diff
@@ -2717,72 +2789,78 @@ \subsubsection{INTERSECT}
     2m = 2km
 \end{verbatim}
 
-\subsection{Common table expressions}
-\label{sec:common-table}
+\subsubsection{Operands}
 
-An ADQL service implementation MAY include support for the following optional
-support for common table expressions:
+Operands of any of the set operators can only be \verb:SELECT: queries.
+Unless if within parentheses, such query can not use any \verb:ORDER BY: or
```

Comment:
Suggest `Unless within parentheses, such queries`

Reply:
Ok. I'll fix it.